.github/workflows/docker.yml

```diff
@@ -3,13 +3,41 @@ on:
   push:
     tags:
     - v*
+    branches:
+    - master
+
 jobs:
-  build:
+  # Builds and pushes the Docker image to Docker Hub and ECR
+  push-to-registries:
     runs-on: ubuntu-latest
     steps:
-    - name: Set output
-      id: vars
-      run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+    - name: Prepare tags
+      id: tags
+      shell: python
+      run: |
+        import datetime
+
+        timestamp = str(int(datetime.datetime.now().timestamp()))
+        base_ecr_tag = '${{ secrets.AWS_ACCOUNT_ID }}.dkr.ecr.eu-central-1.amazonaws.com/tx-mining-service:'
+        base_dockerhub_tag = 'hathornetwork/tx-mining-service:'
```

Comment:
we could also take the chance to make this more explicit, which I think is a great bonus: `base_dockerhub_tag = 'docker.io/hathornetwork/tx-mining-service:'`

Reply:
Will do

Reply:
Thanks!